util/Migrator/DbScripts/2023-01-04_00_Send_Add_CipherId.sql

```diff
@@ -0,0 +1,158 @@
+-- Add CipherId column
+IF COL_LENGTH('[dbo].[Send]', 'CipherId') IS NULL
+BEGIN
+    ALTER TABLE [dbo].[Send] 
+        ADD [CipherId] UNIQUEIDENTIFIER NULL,
+        CONSTRAINT [FK_Send_Cipher] FOREIGN KEY ([CipherId]) REFERENCES [dbo].[Cipher]([Id]);
+END
+GO
+
+-- Recreate View
+IF OBJECT_ID('[dbo].[SendView]') IS NOT NULL
+BEGIN
+    DROP VIEW [dbo].[SendView]
+END
+GO
+
+CREATE VIEW [dbo].[SendView]
+AS
+SELECT
+    *
+FROM
+    [dbo].[Send]
+GO
```

Comment:
Should use `CREATE OR ALTER` syntax, https://contributing.bitwarden.com/contributing/code-style/sql#creating-or-modifying-a-view